.github/workflows/scripts/formatScan/clangtidy.sh

```diff
@@ -2,12 +2,19 @@
 
 source /neural-speed/.github/workflows/scripts/change_color.sh
 
-pip install cpplint
+pip install cmake ninja clang-tidy
```

Comment:
Should we use 14 to match the clang-format we are using?

Reply:
I tried different version, currently no difference of output. I will prefer clang16 since amx-fp16 is supported in this version .
afterall. it is an open question. let's see what others think.

Reply:
Make sense. Maybe stick to 16 to prevent unexpected changes in future versions?

Reply:
sticked to 16.0.4 in [09eda9d](https://github.com/intel/neural-speed/pull/29/commits/09eda9d4e8ebfb76dbb4f5ff65ee439c984177b6)